Source/Scene/Expression.js

```diff
@@ -226,6 +281,33 @@ define([
         return this._value;
     };
 
+    Node.prototype._evaluateVariableString = function(feature) {
+        var exp = this._value;
+        var result = "";
+        var i = exp.indexOf('${');
```

Comment:
Not sure that I would try the regex engine performance here, but could be fine since I don't expect it to be a hotspot for common use cases.